tests/testthat/_snaps/values.md

```diff
@@ -40,6 +40,54 @@
     Code
       value_expected <- c(NaN, -Inf, log10(1:3))
 
+# sequences - doubles
+
+    Code
+      value_seq(test_param_3, 1)
+    Condition
+      Warning:
+      The `default` argument of `new_quant_param()` is deprecated as of dials 1.0.0.
+      i This uses the `default` value of the parameter.
+      i See the Details section of `?value_seq`.
+    Output
+      [1] 0.4
+
+# sequences - integers
+
+    Code
+      value_seq(test_param_1, 1)
+    Condition
+      Warning:
+      The `default` argument of `new_quant_param()` is deprecated as of dials 1.0.0.
+      i This uses the `default` value of the parameter.
+      i See the Details section of `?value_seq`.
+    Output
+      [1] 3
+
+# sequences - character
+
+    Code
+      value_seq(test_param_5, 1)
+    Condition
+      Warning:
+      The `default` argument of `new_qual_param()` is deprecated as of dials 1.0.0.
+      i This uses the `default` value of the parameter.
+      i See the Details section of `?value_seq`.
+    Output
+      [1] "c"
+
+# sequences - logical
+
+    Code
+      value_seq(prune(), 1)
+    Condition
+      Warning:
+      The `default` argument of `new_qual_param()` is deprecated as of dials 1.0.0.
+      i This uses the `default` value of the parameter.
+      i See the Details section of `?value_seq`.
+    Output
+      [1] TRUE
+
```

Comment:
For the tests:

- Update the tests that use `value_seq(, n = 1)` to whatever their new value is (it is most likely the minimum value of the range, or the first value of `values`). Or if you want you can trim these tests down to 1 or 2 tests that just check that `value_seq()` can handle the `n = 1` case.
- Remove any tests that call `new_qual_param()` or `new_quant_param()` directly that also set the `default` argument
- Add a test for both `new_qual_param()` and `new_quant_param()` that ensure that a warning is thrown if `default` is set, and it should check that the `$default` slot of the returned parameter object is `NULL`. If you want, you can put these in `test-deprecated.R`